### PR TITLE
Additional userdata

### PR DESCRIPTION
**Pull request by limed**
Adds additional user data to reflect the ldap group names a global
admin, sudo user and regular user is in.

### CODE REVIEW

**Comment by limed** [comment]
Fixes #12

**Comment by gozer** [comment]
Unless I misremember, isn't the idea to simplify this further by having just 2 group types from an instance point of view?

NUBIS_SUDO_GROUPS
NUBIS_SHELL _GROUPS

Or something like that.

@jd?

**Comment by gozer** [comment]
@tinnightcap rather

**Comment by limed** [comment]
@gozer that was not what I recall in our conversation but I am happy to prefix everything with NUBIS_

**Comment by gozer** [comment]
No, what I mean is that from the instance's point of view (and the puppet code as a side-effect), there are really just 2 categories of groups, the ones that get sudo, and the ones that don't.

Right now, that would translate into:

```
NUBIS_SUDO_GROUPS="global-admins,sudo-users"
NUBIS_USER_GROUPS="users"
```

To me, the logic goes like this, but correct me if I am wrong:
 - TF is told about ldap groups for platform admin purposes, sudo purposes,and non-sudo purposes
 - TF tells lambda about just a list of ldap groups to mirror in Consul
 - TF picks which of these to pass into an instance at launch time (i.e. jumphost vs fluent collector)
 - user-data maps a subset of these groups into just 2 categories (sudo/non-sudo) for confd/puppet to use

So a jumphost, for instances, just needs to know:

>  "These 2 groups from Consul need sudo, these 3 groups from Consul just need local accounts"

Or maybe I am missing something obvious here?

**Comment by tinnightcap** [comment]
i think @gozer is spot on. There only need to be two groups on the unix nodes, sudo and non-sudo.

**Comment by limed** [comment]
@gozer @tinnightcap while your method is probably fine but here is the problem I see. Say we have sane defaults for `NUBIS_SUDO_GROUPS` which has the global-admin ldap group which has us all, and say if me Joe Somebody comes along and deploys an app and says `NUBIS_SUDO_GROUPS` should just be my team and overwrites it. This could potentially lock us out of an account, so if thats ok then its fine.

Of course you could always say to folks please make sure you keep global-admin group in order to not lock us out but I rather be sure about it then not have people have that choice.

**Comment by tinnightcap** [comment]
@limed yea, I think it is just fine for people to have the ability to lock us out of accounts. I agree with you that we can recommend they leave us in the sudoers group, or add us in case of troubleshooting. Generally however, I think it is fine for them to remove us if they choose.

**Comment by limed** [comment]
Fix per recommendation

File: worker/variables.tf [hunk 1]
```diff
@@ -97,3 +97,11 @@ variable "scale_up_load" {
 variable "scale_down_load" {
   default = ""
 }
+
+variable "nubis_sudo_groups" {
+  default = "nubis_global_admins"
```
Comment (limed) [inline worker/variables.tf]: This is an actual ldap group name

**Pull request merged**